epersongroups.md

```diff
@@ -0,0 +1,303 @@
+          },
+          "netid": null,
+          "lastActive": "2019-09-25T15:59:28.000+0000",
+          "canLogIn": true,
+          "email": "user@institution.edu",
+          "requireCertificate": false,
+          "selfRegistered": false,
+          "groups": null,
+          "type": "eperson",
+          "_links": {
+            "self": {
+              "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/epersons/a6086b34-3918-45b7-8ddd-9329a702a26a"
+            }
+          }
+        }
+      ]
+    },
+    "_links": {
+      "self": {
+        "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42/epersons"
+      }
+    },
+    "page": {
+      "number": 0,
+      "size": 10,
+      "totalPages": 1,
+      "totalElements": 10
+    }
+}
+```
+
+### Changing Epeople in a Group
+**PUT /api/eperson/groups/<:groupuuid>/epersons**
+
+_Unsupported._ At this time, we do not support removing or replacing all epersons in a single request. Please use `DELETE /api/eperson/groups/<:groupuuid>/epersons/<:epersonuuid>` to remove the epersons one by one.
+
+### Add an EPerson to a parent Group
+**POST /api/eperson/groups/<:groupuuid>/epersons**
+
+To add an eperson to a parent group, perform a POST to the Epeople of a Group endpoint when logged in as admin.
+
+The actual eperson is part of the body using the uri-list
+
+Example:
+```bash
+curl -i -X POST "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/epersons"
+  -H "Content-Type:text/uri-list"
+  -d "https://dspace7.4science.it/dspace-spring-rest/api/eperson/epersons/a6086b34-3918-45b7-8ddd-9329a702a26a"
+```
+
+The eperson is mandatory
+
+Return codes:
+* 204: if the update succeeded 
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions 
+* 422: if the specified eperson is not found
+
+### Remove an EPerson from a parent Group
+**DELETE /api/eperson/groups/<:groupuuid>/epersons/<:epersonuuid>**
+
+To remove an eperson from a parent group, perform a DELETE to the Epeople of a Group endpoint when logged in as admin.
+
+Example:
+```bash
+curl -i -X DELETE 
+  "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/epersons/a6086b34-3918-45b7-8ddd-9329a702a26a"
+```
+
+The above request would remove the mapping between the group with UUID `617cf46b-535c-42d5-9d22-327ce2eff6dc`
+ and the eperson with UUID `a6086b34-3918-45b7-8ddd-9329a702a26a`.
+
+
+Return codes:
+* 204: if the update succeeded (including the case of no-op if the eperson was not a member)
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions 
+* 404 Not found - if the parent group doesn't exist
+* 422 Unprocessable Entity - if the child group doesn't exist
+* 422 Unprocessable Entity - if the specified eperson doesn't exist
```

Comment:
just a style note, we usually combine the different scenarios in a single sentence

Reply:
This has been adjusted now